use_default_vars/use_default_vars.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+ansible-playbook -i inventory/localhost ./use_default_vars.yml
```

Comment:
Could you create a README.md that explains what is being demonstrated here and how to run the playbook? [Here's an example of doing this](https://github.com/ansible/ansible-examples/blob/master/lamp_haproxy/README.md)

Also, since this is such a simple playbook to run it's probably not necessary to have a bash script wrapping `ansible-playbook`. Would you mind removing this file?

Thanks for the contribution!